### ISSUE TITLE
BUG - Path must be a string. Received undefined

### ISSUE THREAD

**Issue by reisel** [stripe/stripe-node#436]
Node version 8.9.4
Stripe version: 5.5.0

when creating a plan I get the error.
the reason is that the create spec has no path.
then when calling createResourcePathWithSymbols(spec.path) from StripeMethod
createResourcePathWithSymbols is called with undefined.

inside createResourcePathWithSymbols the method calls path.join with the undefined value and this throws the exception.

**Comment by ob-stripe** [comment]
Hi @reisel, thanks for the report. Can you provide more information about this issue? If you could provide sample code to reproduce the issue, it would be very helpful.

**Comment by reisel** [comment]
@ob-stripe 
sure.
```javascript
class StripeChargeRequest {
	constructor({ amount, currency, source }) {
		this.amount = amount
		this.currency = currency
		this.source = source
	}
	
	toJSON() {
		return {
			amount: this.amount,
			currency: this.currency,
			source: this.source
		}
	}
}
const stripe = require('stripe')('sk_test_xxx')
const request = new StripeChargeRequest({ amount: 1200, currency: 'USD', source: 'tok_visa' })
stripe.plans.create(request)
```

please note,
I know the stripe library wasn't designed to support the custom class.
but still in this case there is an error message that need to be displayed:

``` javascript
err = new Error(
          'Stripe: Unknown arguments (' + args + '). Did you mean to pass an options ' +
          'object? See https://github.com/stripe/stripe-node/wiki/Passing-Options.' +
          ' (on API request to ' + requestMethod + ' `' + path + '`)'
        );
```

**Comment by ob-stripe** [comment]
Ah, gotcha. Indeed, it appears that for API methods where `spec.path` is undefined, the code crashes while trying to format the error message.

**Comment by ob-stripe** [comment]
Fixed in 5.6.1. Thanks for the contribution @reisel!
